### PR TITLE
Internal audit/easy fixes alex

### PR DESCRIPTION
**Pull request by acollette**
easy fixes for Guardian contracts

### CODE REVIEW

File: lib/forge-std
Comment (Thomas-Smets) [inline lib/forge-std:1]: This might give issues with other PRs

File: test/fuzz/guardians/FactoryGuardian/UnPauseWithoutArgs.fuzz.t.sol
Comment (Thomas-Smets) [inline test/fuzz/guardians/FactoryGuardian/UnPauseWithoutArgs.fuzz.t.sol:1]: Why is this file removed?

File: src/guardians/RegistryGuardian.sol
Comment (Thomas-Smets) [inline src/guardians/RegistryGuardian.sol:1]: I think you have overwritten Zeki's previous PR here.

File: src/guardians/FactoryGuardian.sol [hunk 1]
```diff
@@ -24,13 +24,7 @@ abstract contract FactoryGuardian is BaseGuardian {
...
     ////////////////////////////////////////////////////////////// */
 
-    event PauseUpdate(bool createPauseUpdate);
-
-    /*
-    //////////////////////////////////////////////////////////////
-                            ERRORS
-    //////////////////////////////////////////////////////////////
-    */
+    event PauseFlagUpdated(bool createPauseUpdate);
```
Comment (Thomas-Smets) [inline src/guardians/FactoryGuardian.sol:27]: Should we rename it to PauseFlagsUpdated, even her there is only one flag, to be consistent with the other guardians?

File: src/guardians/FactoryGuardian.sol [hunk 2]
```diff
@@ -10,7 +10,7 @@ import { BaseGuardian } from "./BaseGuardian.sol";
 /**
  * @title Factory Guardian
  * @author Pragma Labs
- * @notice This module provides the logic for the Factory that allows authorized accounts to trigger an emergency stop.
+ * @notice This module holds the logic that allows authorized accounts to trigger an emergency stop in the Factory.
```
Comment (Thomas-Smets) [inline src/guardians/FactoryGuardian.sol:13]:
```suggestion
 * @notice Logic inherited by the Factory that allows an authorized guardian to trigger an emergency stop.
```

File: src/guardians/RegistryGuardian.sol
Comment (acollette) [reply src/guardians/RegistryGuardian.sol:1]: adapted based on Zeki's PR

File: src/guardians/FactoryGuardian.sol [hunk 3]
```diff
@@ -39,55 +33,41 @@ abstract contract FactoryGuardian is BaseGuardian {
...
-        createPaused = createPaused && createPaused_;
-
-        emit PauseUpdate(createPaused);
+    function unpause(bool createPaused_) external onlyOwner {
+        emit PauseUpdated(createPaused = createPaused && createPaused_);
     }
 
     /**
-     * @inheritdoc BaseGuardian
+     * @notice This function is not implemented. If someone would ever need to call this function, it means
```
Comment (j-vp) [inline src/guardians/FactoryGuardian.sol:69]: Change comment, remove that the protocol can't be trusted anymore. 'No reason to create be able to create an Account' to be edited.

unpause() got an OnlyOwner here vs the public function of the base, I would just revert with a FunctionNotImplemented

File: test/fuzz/guardians/FactoryGuardian/Pause.fuzz.t.sol [hunk 4]
```diff
@@ -68,7 +68,7 @@ contract Pause_FactoryGuardian_Fuzz_Test is FactoryGuardian_Fuzz_Test {
         // When: the Guardian pauses.
         vm.startPrank(users.guardian);
         vm.expectEmit();
```
Comment (j-vp) [inline test/fuzz/guardians/FactoryGuardian/Pause.fuzz.t.sol:70]: add a (true, true, true, true) here, otherwise I don't think it actually checks the content of the event

File: src/guardians/BaseGuardian.sol [hunk 5]
```diff
@@ -88,11 +86,11 @@ abstract contract BaseGuardian is Owned {
     function pause() external virtual onlyGuardian { }
 
     /**
-     * @notice This function is used to unPause all flags.
+     * @notice This function is used to unpause all flags.
      * @dev If the protocol is not unpaused after 30 days, any user can unpause the protocol.
      * This ensures that no rogue owner or guardian can lock user funds for an indefinite amount of time.
      * All users have now at least a two-day window to withdraw assets and close positions before
-     * the protocol can again be paused (after 32 days).
+     * the protocol can again be paused 32 days after the contract was previously paused.
```
Comment (zekiblue) [inline src/guardians/BaseGuardian.sol:93]:
```suggestion
     * The Protocol can be paused again 32 days after the previous pause.
```

File: test/fuzz/guardians/FactoryGuardian/Pause.fuzz.t.sol (hunk 4 above)
Comment (acollette) [reply test/fuzz/guardians/FactoryGuardian/Pause.fuzz.t.sol:70]: vm.expectEmit() checks all elements by default (same as true,true,true,true)

File: src/guardians/FactoryGuardian.sol [hunk 6]
```diff
@@ -39,55 +39,42 @@ abstract contract FactoryGuardian is BaseGuardian {
...
-        createPaused = createPaused && createPaused_;
-
-        emit PauseUpdate(createPaused);
+    function unpause(bool createPaused_) external onlyOwner {
+        emit PauseUpdated(createPaused = createPaused && createPaused_);
     }
 
     /**
-     * @inheritdoc BaseGuardian
+     * @notice This function is not implemented. No reason to be able to create an Account if the owner of the Factory is not able to unpause createAccount().
```
Comment (Thomas-Smets) [inline src/guardians/FactoryGuardian.sol:75]:
```suggestion
     * @notice This function is not implemented. No reason to be able to create an Account if the owner of the Factory did not unpause createAccount().
```

**Review by Thomas-Smets — APPROVED** [review]
gtg after comment

File: test/fuzz/Factory/TransferFrom.fuzz.sol [hunk 7]
```diff
@@ -37,6 +38,28 @@ contract TransferFrom_Factory_Fuzz_Test is Factory_Fuzz_Test {
...
+        vm.assume(nonOwner != address(mockOracles.stable2ToUsd));
+        vm.assume(nonOwner != address(mockOracles.token1ToUsd));
+        vm.assume(nonOwner != address(mockOracles.token2ToUsd));
+        vm.assume(nonOwner != address(mockOracles.token3ToToken4));
+        vm.assume(nonOwner != address(mockOracles.token4ToUsd));
+        vm.assume(nonOwner != address(mockOracles.nft1ToToken1));
+        vm.assume(nonOwner != address(mockOracles.nft2ToUsd));
+        vm.assume(nonOwner != address(mockOracles.nft3ToToken1));
+        vm.assume(nonOwner != address(mockOracles.sft1ToToken1));
+        vm.assume(nonOwner != address(mockOracles.sft2ToUsd));
```
Comment (zekiblue) [inline test/fuzz/Factory/TransferFrom.fuzz.sol:62]: This will contradict with other pr, better to delete maybe

**Review by zekiblue — APPROVED** [review]
lgtm

File: test/fuzz/Factory/TransferFrom.fuzz.sol (hunk 7 above)
Comment (j-vp) [reply test/fuzz/Factory/TransferFrom.fuzz.sol:62]: it's in dev already so shouldn't be a change
